packages/neos-ui-i18n/src/model/TranslationRepository.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * This file is part of the Neos.Neos.Ui package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+import {Locale} from './Locale';
+import {Translation, type TranslationDTO} from './Translation';
+import type {TranslationAddress} from './TranslationAddress';
+
+export type TranslationsDTO = Record<string, Record<string, Record<string, TranslationDTO>>>;
+
+export class TranslationRepository {
+    private _translationsByAddress: Record<string, null | Translation> = {};
```

Comment:
a little nitpick: i  think the underscore is obsolete in combination with the `private` visibility. Even its js i like to refer to psr12 which denotes that underscores should not be used to denote visibility.